plugins/source/aws/README.md

```diff
@@ -0,0 +1,330 @@
+        local_profile: <NAMED_PROFILE>
+        // Optional. Specify the Role Session name
+        role_session_name: ""
+      - id: <AccountID_Alias_2>
+        local_profile: provider
+        // Optional. Role ARN we want to assume when accessing this account
+        role_arn: <YOUR_ROLE_ARN_2>
+```
+
+#### Arguments for Accounts block
+
+- `role_arn` **(Optional)** - The role that CloudQuery will use to perform the fetch
+- `local_profile` **(Optional)** - Local Profile is the named profile in your shared configuration file (usually `~/.aws/config`) that you want to use for the account
+- `external_id` **(Optional)** - The unique identifier used by non-AWS entities to assume a role in an AWS account
+- `role_session_name` **(Optional)** - Override the default Session name.
+- `default_region` **(Optional)** - this sets the Default Region for the AWS SDK. If you are assuming a role in a partition other than the AWS commercial region, it is important that this attribute is set
+- `regions` **(Optional)** - Limit fetching resources within this specific account to only these regions. This will override any regions specified in the provider block. You can specify all regions by using the `*` character as the only argument in the array
+
+### Assume Role with MFA
+
+In order to assume role with MFA, you need to request temporary credentials using STS "get-session-token".
+
+```bash
+aws sts get-session-token --serial-number <YOUR_MFA_SERIAL_NUMBER> --token-code <YOUR_MFA_TOKEN_CODE> --duration-seconds 3600
+```
+
+Then export the temporary credentials to your environment variables.
+
+```bash
+export AWS_ACCESS_KEY_ID=<YOUR_ACCESS_KEY_ID>
+export AWS_SECRET_ACCESS_KEY=<YOUR_SECRET_ACCESS_KEY>
+export AWS_SESSION_TOKEN=<YOUR_SESSION_TOKEN>
+```
+
+## Query Examples
+
+### Find all public facing load balancers
+
+```sql
+SELECT * FROM aws_elbv2_load_balancers WHERE scheme = 'internet-facing';
+```
+
+### Find all non encrypted RDS instances
+
+```sql
+SELECT * FROM aws_rds_clusters WHERE storage_encrypted IS FALSE;
+```
+
+### Find all s3 buckets that are able to be public
+
+```sql
+SELECT arn, region
+FROM aws_s3_buckets
+WHERE block_public_acls IS NOT TRUE
+    OR block_public_policy IS NOT TRUE
+    OR ignore_public_acls IS NOT TRUE
+    OR restrict_public_buckets IS NOT TRUE
+
+```
+
+## Building the Provider
+
+```bash
+make build
+```
+
+Running Provider locally:
+
+1. Clone repository to local machine
+
+2. [Optional] Start a local database:
+   ```bash
+   make pg-start
+   ```
+3. [Optional] Configure the `cloudquery.yml`
+   ```bash
+   make os=Linux arch=arm64 install
+   ./cloudquery init aws
+   ```
+4. Start the provider in a different tab/session
+   ```bash
+   make run
+   ```
+5. Execute CloudQuery Fetch using the locally built provider
+   ```bash
+   make fetch
+   ```
+
+More information can be found in the [CloudQuery documentation](https://docs.cloudquery.io/docs/developers/debugging)
```

Comment:
```suggestion
## Building the Provider

```bash
make build
```

Running Provider locally:

1. Clone repository to local machine

2. [Optional] Start a local database:
   ```bash
   make pg-start
   ```
3. [Optional] Configure the `cloudquery.yml`
   ```bash
   make os=Linux arch=arm64 install
   ./cloudquery init aws
   ```
4. Start the provider in a different tab/session
   ```bash
   make run
   ```
5. Execute CloudQuery Fetch using the locally built provider
   ```bash
   make fetch
   ```

More information can be found in the [CloudQuery documentation](https://docs.cloudquery.io/docs/developers/debugging)
```

Maybe remove this for now?